infinite-slider.js

```diff
@@ -208,6 +208,18 @@
                 interactionCurrent.y += dy;
               }
             }
+            // had to add this to avoid getting undefined at line 211 when clicking google maps
+            // this seems like a hack, if you know of a better solution please delete this.
+            // *** start of added code
+               if (!interactionStart && !interactionCurrent) {
+                  interactionStart = interactionCurrent = { x: 0};
```

Comment:
I created [this](https://github.com/gilbox/kinetic-slider) for that reason but yeah it would be nice to add that as an option. A PR for that feature would be welcome ( ◠‿◠ )